server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -108,7 +108,7 @@
         "socketIo": {
             "perMessageDeflate": true
         },
-        "sessionStickinessDurationMs": 3600000
+        "sessionStickinessDurationMs": 10800000
```

Comment:
I think we can leave it as it is in OSS. We can override this in AFR.

Reply:
Could I know what you mean by AFR? Where do we override it? Thanks.

Reply:
Azure fluid relay :) In the cluster specific param files

Reply:
Sounds good! Done.